recipes/assimp/5.x/conanfile.py

```diff
@@ -141,17 +149,18 @@ def _depends_on_zlib(self):
 
     def requirements(self):
         # TODO: unvendor others libs:
-        # - clipper (required by IFC importer): can't be unvendored because CCI
-        #   has 6.4.2, not API compatible with 4.8.8 vendored in assimp
         # - Open3DGC
-        # - openddlparser
         if tools.Version(self.version) < "5.1.0":
             self.requires("irrxml/1.2")
         else:
             self.requires("pugixml/1.11")
+            self.requires("openddl-parser/cci.20211217")
 
+        self.requires("zlib/1.2.11")
```

Comment:
```suggestion
        self.requires("zlib/1.2.11")
```

Not a direct dependency as far as I remember. It's a dependency of minizip